_health-care/_js/actions/index.js

```diff
@@ -15,3 +17,18 @@ export function veteranUpdateField(propertyPath, value) {
     value
   };
 }
+
+export function sectionCompleteUpdateField(path) {
+  return {
+    type: SECTION_COMPLETE_UPDATE_FIELD,
+    path
+  };
+}
+
+export function sectionReviewStatusUpdateField(propertyPath, value) {
+  return {
+    type: SECTION_REVIEW_STATUS_UPDATE_FIELD,
+    propertyPath,
```

Comment:
I think this should be called `path`, since it takes the whole path (e.g., `personal-information/name-and-general-info`) and not the `propertyPath` (e.g., `name-and-general-info`), which is what `state.veteran` uses.